auth/auth0_manager.go

```diff
@@ -71,12 +95,25 @@ func (a *Auth0Manager) Authenticate(username, password string) (bool, error) {
 	}
 
```

Comment:
Again, I think this is over-complicated. For now, we only need to throttle before making the auth0 call:
```
<-a.throttle
if err := a.client.Post("/oauth/ro", req, nil); err != nil { ... }
```
This is the **only** place we should deal with throttling, since it's only the auth0 api giving us the 429 response.

Assuming this is enough to prevent the 429 response, then we don't need all of this extra backoff/retry logic. 
Let's keep it simple for now.